ajax/libs/tv4/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "tv4",
+  "filename": "tv4.min.js",
+  "version": "1.2.7",
+  "author": "Geraint Luff",
+  "description": "A public domain JSON Schema validator for JavaScript",
+  "keywords": [
+    "json-schema",
+    "schema",
+    "validator",
+    "tv4"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/geraintluff/tv4.git"
+  },
+  "homepage": "https://geraintluff.github.io/tv4/",
+  "licenses": [
+    "Public Domain",
```

Comment:
@dannyxx001 `Public Domain` doesn't exist in [license-list](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json). You need to provide `url` for the license.